_includes/scripts.html

```diff
@@ -8,6 +8,55 @@
 {% endfor %} {% elsif layout.custom_js %} {% for js_file in layout.custom_js %}
 <script src="/assets/js/{{ js_file }}.js" type="text/javascript"></script>
 {% endfor %} {% endif %} {% if site.gtm_id %}
+<script src="{{ site.assets_base_url }}js/asciinema-player.min.js"></script>
+
+<script>
+  AsciinemaPlayer.create(
+    "{{ site.assets_base_url }}asciinema/Patcher.cast",
+    document.getElementById("patcher-player-cast-1"),
+    { cols: 100, rows: 20, autoPlay: true }
+  );
+  AsciinemaPlayer.create(
+    "{{ site.assets_base_url }}asciinema/Patcher.cast",
+    document.getElementById("patcher-player-cast-2"),
+    { cols: 101, rows: 24, autoPlay: true }
```

Comment:
Was this one done on purpose with different cols and rows? Any others have the same like "151" and "36"